Src/BootCamp.Chapter/Shop.cs

```diff
@@ -64,7 +75,16 @@ public decimal Buy(Item item)
         /// </returns>
         public Item Sell(string item)
         {
-            return null;
+            var soldItems = new List<Item>(); 
+            for (int i = 0; i < Items.Count; i++)
+            {
+                if (item == Items[i].Name)
+                {
+                    soldItems.Add(Items[i]); 
```

Comment:
Not sure if you've learnt this but you can add the `break;` keyword to exit the for loop once an item is found. 

If you've not learnt it yet you can disregard this comment :)

Reply:
we learned that in chapter2 so I can changed that. Then I also think I do not need the soldItem variable. I will see

Reply:
moment, if I use a break how can I then return the item found. Here I cannot follow you complete.

Reply:
or do you mean something like this : 

```
 public Item Sell(string item)
        {
            for (int i = 0; i < Items.Count; i++)
            {
                if (item == Items[i].Name)
                {
                    Money += Items[i].Price; 
                    return Items[i]; 
                }
            }
            return null; 
            
        }
```

Reply:
What you've written here, to return instead of break, is probably better. Go with that :).